code/game/objects/items/stacks/medical.dm

```diff
@@ -85,6 +85,9 @@
 					continue
 				if(used == amount)
 					break
+				if(W.is_surgical() && user.a_intent == I_HELP)
+					to_chat(user, "<span class='notice'>You refrain from closing the incision.</span>")
```

Comment:
I feel like this might work better as an input, might make things a little more clear for newer players. Either that or an addition to the message like "you refrain from closing the incision as your intent is set to help."